src/main/java/net/mcreator/ui/modgui/VillagerProfessionGUI.java

```diff
@@ -0,0 +1,203 @@
+import net.mcreator.workspace.elements.ModElement;
+
+import javax.swing.*;
+import java.awt.*;
+import java.io.File;
+import java.util.Collections;
+import java.util.Locale;
+
+public class VillagerProfessionGUI extends ModElementGUI<VillagerProfession> {
+
+	private final VTextField name = new VTextField(30);
+	private final MCItemHolder pointOfInterest = new MCItemHolder(mcreator, ElementUtil::loadBlocks);
+	private final SoundSelector actionSound = new SoundSelector(mcreator);
+	private final VComboBox<String> professionTextureFile = new SearchableComboBox<>();
+
+	private final int fact = 64 * 5;
+
+	private final JLabel clo = new JLabel();
+	private final ValidationGroup page1group = new ValidationGroup();
+
+	public VillagerProfessionGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
+		super(mcreator, modElement, editingMode);
+		this.initGUI();
+		super.finalizeGUI();
+	}
+
+	@Override protected void initGUI() {
+		professionTextureFile.setRenderer(new WTextureComboBoxRenderer.OtherTextures(mcreator.getWorkspace()));
+		professionTextureFile.setPrototypeDisplayValue("XXXXXXXXXXXXXXXXXXXXXXXXXX");
+
+		JPanel panel2 = new JPanel(new BorderLayout());
+
+		ComponentUtils.deriveFont(name, 16);
+
+		JPanel subpanel = new JPanel(new GridLayout(4, 2, 0, 2));
+		subpanel.setOpaque(false);
+
+		name.setEnabled(false);
+
+		ComponentUtils.deriveFont(name, 16);
+
+		subpanel.add(HelpUtils.wrapWithHelpButton(this.withEntry("villagerprofessions/name"),
+				L10N.label("elementgui.villager_profession.name")));
+		subpanel.add(name);
+
+		subpanel.add(HelpUtils.wrapWithHelpButton(this.withEntry("villagerprofessions/point_of_interest"),
+				L10N.label("elementgui.villager_profession.point_of_interest")));
+		subpanel.add(PanelUtils.centerInPanel(pointOfInterest));
+
+		subpanel.add(HelpUtils.wrapWithHelpButton(this.withEntry("villagerprofessions/action_sound"),
+				L10N.label("elementgui.villager_profession.action_sound")));
+		subpanel.add(actionSound);
+
+		professionTextureFile.addActionListener(e -> updateProfessionTexturePreview());
+
+		JButton importProfessionTexture = new JButton(UIRES.get("18px.add"));
+		importProfessionTexture.setToolTipText(L10N.t("elementgui.villager_profession.import_profession_texture"));
+		importProfessionTexture.setOpaque(false);
+		importProfessionTexture.addActionListener(e -> {
+			TextureImportDialogs.importMultipleTextures(mcreator, TextureType.VILLAGER_PROFESSION);
+			professionTextureFile.removeAllItems();
+			professionTextureFile.addItem("");
+			mcreator.getFolderManager().getTexturesList(TextureType.VILLAGER_PROFESSION)
+					.forEach(el -> professionTextureFile.addItem(el.getName()));
+		});
+
+		subpanel.add(HelpUtils.wrapWithHelpButton(this.withEntry("villagerprofessions/profession_texture"),
+				L10N.label("elementgui.villager_profession.profession_texture")));
+		subpanel.add(PanelUtils.centerAndEastElement(professionTextureFile, importProfessionTexture, 0, 0));
```

Comment:
`0, 0` is not needed here as 0 is the default value used.
```suggestion
		subpanel.add(PanelUtils.centerAndEastElement(professionTextureFile, importProfessionTexture));
```